### PR TITLE
User login branch

### PR DESCRIPTION
**Pull request by YatKat**

### CODE REVIEW

File: src/main/java/mate/academy/springdemo/dto/input/UserRegistrationInput.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserRegistrationInput {
+    private String login;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/dto/input/UserRegistrationInput.java:11]: maybe registration and input DTO should be in different some way?

File: src/main/java/mate/academy/springdemo/controller/AuthController.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    @RequestMapping(value = "/login", method = RequestMethod.POST)
+    public String login(UserLoginInput userLoginInput){
+        log.info("login controller is working");
+        return "/";
+    }
+
+    @RequestMapping(value = "/register", method = RequestMethod.GET)
+    public String register(){
+        log.info("register controller is working");
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/controller/AuthController.java:29]: I see that you haven't done any implementation of those methods, but I don't know whether we should do that either

**Review by ghost — COMMENTED** [review]
LGTM

File: src/main/java/mate/academy/springdemo/dto/input/UserRegistrationInput.java (hunk 1 above)
Comment (YatKat) [reply src/main/java/mate/academy/springdemo/dto/input/UserRegistrationInput.java:11]: Occama's razor: there is no need to multiply enough. I thing two fields it is enough for both variants.

**Review by boroda4436 — CHANGES_REQUESTED** [review]
Log4j2 configuration should be implemented

**Review by boroda4436 — APPROVED** [review]
LGTM

**Pull request merged**